react/LocationSearch.tsx

```diff
@@ -111,20 +115,31 @@ const LocationSearch: React.FC<LocationSearchProps> = ({
     searchTerm,
     DEBOUNCE_DELAY_IN_MS
   )
-  const [suggestions, loadingSuggestions] = useSuggestions(debouncedSearchTerm)
+  const {
+    data: sessionTokenData,
+    loading: loadingSessionToken,
+    refetch: refetchSessionToken,
+  } = useQuery<Query, {}>(SESSION_TOKEN)
```

Comment:
you should probably use the `notifyOnNetworkStatusChange` option here if you want the `loadingSessionToken` to be `true` after you call `refetchSessionToken`